app/models/vm_or_template.rb

```diff
@@ -187,6 +187,8 @@ class VmOrTemplate < ApplicationRecord
   alias_method :parent_cluster, :ems_cluster
   alias_method :owning_cluster, :ems_cluster
 
+  supports_not :resize, :reason => _("Resize Operation is not available for VM or Template.")
```

Comment:
as this is unlikely to be called on `VmOrTemplate` and its not really a reason other than it's unsupported, I would drop the reason.